### DISCUSSION TITLE
DistributionTool fails to build with PNGs as Assets

### DISCUSSION THREAD

**Discussion by erikjmurray** [elgatosf/elgato#79] [Help] [answered]
I refactored the directory structure of the StreamDeck template into my own base project. Using that base `template`, I created my own plugin `project`. I created a symlink to the root folder for development which works correctly on the StreamDeck and now I am trying to use the DistributionTool to finalize the plugin. 

The distribution works for the refactored base template with SVG assets using:

`.\DistributionTool.exe -b -i com.mydomain.template.sdPlugin -o FinalizedPlugins`. 

However, I decided to use PNGs for my assets and I get the following error when I run the command referencing my project directory. 

```PowerShell
Plugins> .\DistributionTool.exe -b -i com.mydomain.project.sdPlugin -o FinalizedPlugins
                    DistributionTool        using printf
                    DistributionTool        Validating and exporting the plugin com.mydomain.project.sdPlugin
                    DistributionTool        Error: The plugin has an icon "pluginIcon@2x.png" that does not exist.
                    DistributionTool        Error: The plugin is invalid
```

I found a similar issue in [discussion 76](https://github.com/orgs/elgatosf/discussions/76), but upon further reading I see that the issue is slightly different and the most recent version of either DistributionTool on the website did not work for me.

**Follow up:** I'm able to build the plugin if I convert the PNGs to SVGs. However, the SVGs do not render on the StreamDeck. The SVGs look terrible compared to the original PNGs anyway, this was meant as a proof of concept.

**Comment by ChekTek** [comment, marked as answer]
Hi @erikjmurray,

Sorry the error message provided by the DistributionTool is not very clear, but when using `rasterized` images like `png` you will need to provide two copies of the file. You can find the image resolution details in the plugin [style guide](https://docs.elgato.com/sdk/plugins/style-guide#sizes)

Since `svg` images can scale to fit their container, but `png` images do not, which is why we require a low and high resolution copy.

**Reply by erikjmurray** [reply]
That was it! Thank you! It might be worth adding a note about it on the [Packaging](https://docs.elgato.com/sdk/plugins/packaging) page of your documentation. I never would have known to look for that.
